docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+# Event Broadcast and Subscription
+
+
+The application can broadcast its own events from your application to all listeners who want to listen. The events can either be predefined system [events from the platform](#platform) or user-defined events. In a predefined system event, only platform modules can broadcast the system events. Whereas, in a user-defined event, UI and service applications broadcast the system events.
+
+The main features of the Event API are:
+
+- Event publication
+
+  You can [publish an event](#broadcast) using the [Tizen.Applications.EventManager.ApplicationEventManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.EventManager.ApplicationEventManager.html) class.
+
+- Event subscription
+
+  You can [subscribe to an event](#manage) using the [Tizen.Applications.EventManager.EventReceiver](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.EventManager.EventReceiver.html) class.
+
+- Launch-On-Events
+
+  You can launch the service applications when the desired target [event occurs](#launch).
```

Comment:
Please make this small change in the word that is used to link:

You can [launch](#launch) the service applications when the desired target event occurs.